app/src/main/java/com/iic/shopingo/api/request/OutgoingRequestApiResult.java

```diff
@@ -0,0 +1,23 @@
+package com.iic.shopingo.api.request;
+
+import com.iic.shopingo.api.ApiResult;
+import com.iic.shopingo.api.models.ApiOutgoingRequest;
+import com.iic.shopingo.api.models.converters.OutgoingRequestConverter;
+import com.iic.shopingo.dal.models.OutgoingRequest;
+
+/**
+ * Created by asafg on 11/03/15.
+ */
+public class OutgoingRequestApiResult extends ApiResult {
+
+  public OutgoingRequest request;
+
+  public OutgoingRequestApiResult(ApiOutgoingRequest request) {
+    super();
```

Comment:
No need to call `super` here, constructors implicitly call the super no-arg constructor unless specified differently (by calling a super constructor that receives arguments).